test cases/common/68 number arithmetic/meson.build

```diff
@@ -23,6 +23,8 @@ endif
 
 assert((5 % 2) == 1, 'Integer modulo (odd) is broken')
 assert((4 % 2) == 0, 'Integer modulo (even) is broken')
+assert(2.is_even() == 1, 'int is_even() broken')
+assert(2.is_odd() == 0, 'int is_odd() broken')
```

Comment:
We should also test for the negative case here:

```
assert(3.is_even() == false, 'int.is_even() is broken for odd values')
assert(5.is_odd() == true, 'int.is_odd() is broken for odd values')
```

The original test will pass if `is_even()` always returns `true` and `is_odd()` always returns `false`.